plugins/net_plugin/net_plugin.cpp

```diff
@@ -2412,8 +2416,29 @@ namespace eosio {
                fc_elog( logger, "Error accepting connection: ${m}", ("m", ec.message()));
                // For the listed error codes below, recall start_listen_loop()
                switch (ec.value()) {
+                  case EMFILE: // same as boost::system::errc::too_many_files_open
+                  {
+                     // no file descriptors available to accept the connection. Wait on async_timer
+                     // and retry listening
+                     if (!accept_error_timer) {
+                        try {
+                           accept_error_timer = std::make_unique<boost::asio::deadline_timer>(my_impl->thread_pool->get_executor());
+                        } catch(...) {
+                           elog( "net_plugin_impl::start_listen_loop() - exception when allocating memory" );
```

Comment:
Should be `fc_elog( logger` in `net_plugin`. Feel free to fix the couple of other wrong uses of `elog` and one use of `wlog` in `net_plugin`.

Reply:
how is the `logger` symbol accessible gere since we don't capture `this`?

https://github.com/AntelopeIO/leap/blob/98dea84c0a4ab6319080df48d6d24d7f8561f3a3/plugins/net_plugin/net_plugin.cpp#L3840

Reply:
`logger` is static to the file.

Reply:
oh my bad I missed the closing brace and thought it was in `net_plugin_impl`. Thanks!